sql/moz-fx-data-shared-prod/firefox_desktop_derived/urlbar_events_daily_v1/query.sql

```diff
@@ -0,0 +1,110 @@
+WITH
+  temp_unnested AS (
+  SELECT
+    submission_date,
+    event_id,
+    event_action,
+    res.product_result_type AS product_result_type,
+    normalized_channel,
+    normalized_country_code,
+    pref_fx_suggestions,
+    pref_sponsored_suggestions,
+    is_terminal,
+    CASE
+      WHEN (product_selected_result = res.product_result_type AND event_action = 'engaged' AND is_terminal = TRUE) THEN 1
+    ELSE
+    0
+  END
+    AS is_clicked,
```

Comment:
This is more of a style nit, but leaving these as booleans could improve readability:
```sql
(product_selected_result = res.product_result_type AND event_action = 'engaged' AND is_terminal) AS is_clicked
```